### PR TITLE
Add Cutebot line sensor support

### PR DESCRIPTION
**Pull request by hbradio**
Adds support for the Cutebot line sensors.

### CODE REVIEW

File: inc/drivers/cutebot/CutebotLineSensors.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#ifndef CUTEBOT_LINE_SENSORS_H
+#define CUTEBOT_LINE_SENSORS_H
+
+#include "MicroBitComponent.h"
+#include "MicroBitIO.h"
+
+class CutebotLineSensors : public MicroBitComponent
+{
+    MicroBitIO io;
```
Comment (cabarnes) [inline inc/drivers/cutebot/CutebotLineSensors.h]: Should this be `&io`?

Comment (hbradio) [reply inc/drivers/cutebot/CutebotLineSensors.h]: Yes. I'll fix that in the GigglebotLineSensors.h, too.

Comment (hbradio) [reply inc/drivers/cutebot/CutebotLineSensors.h]: (for the Gigglebot's i2c)

File: source/main.cpp [hunk 2]
```diff
@@ -158,11 +158,11 @@ int main()
     bot = new Gigglebot(uBit.i2c);
     if (!bot->isDetected()) {
         delete bot;
-        bot = new Cutebot(uBit.i2c);
+        bot = new Cutebot(uBit.i2c, uBit.io);
         if (!bot->isDetected()) {
             delete bot;
             uBit.display.scrollAsync("NO BOT");
-	}
+	    }
```
Comment (cabarnes) [inline source/main.cpp]: Seems like there's a tab here causing the indentation to be inconsistent

Comment (hbradio) [reply source/main.cpp]: Good catch. Fixed.

**Review by cabarnes — APPROVED** [review]

**Pull request merged**